### PR TITLE
Display the Clearpay SVG when Locale is set to UK

### PR DESCRIPTION
**Pull request by adamjcampbell**
## Summary of Changes

- Adds Locale to the configuration
- Adds Clearpay SVGs
- Returns the correct drawable based on locale and color scheme

### CODE REVIEW

File: afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt [hunk 1]
```diff
@@ -51,14 +52,12 @@ class AfterpayPriceBreakdown(context: Context, attrs: AttributeSet?) : FrameLayo
...
     private val infoUrl: String
-        get() = when (Afterpay.configuration?.currency?.currencyCode) {
-            "AUD" -> "https://static-us.afterpay.com/javascript/modal/au_rebrand_modal.html"
-            "NZD" -> "https://static-us.afterpay.com/javascript/modal/nz_rebrand_modal.html"
-            "CAD" -> "https://static-us.afterpay.com/javascript/modal/ca_rebrand_modal.html"
-            else -> "https://static-us.afterpay.com/javascript/modal/us_rebrand_modal.html"
+        get() {
+            val locale = Afterpay.configuration?.locale ?: Locale.US
+            val country = locale.country.toLowerCase(Locale.ROOT)
+            return "https://static-us.afterpay.com/javascript/modal/${country}_rebrand_modal.html"
```
Comment (Rypac) [inline afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt:60]: Very nice 👌

**Review by Rypac — APPROVED** [review]
Great work! 🎉

File: afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt (hunk 1 above)
Comment (adamjcampbell) [reply afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt:60]: Now you've reminded me that I should restrict the valid locale set like I have done on iOS

Comment (Rypac) [reply afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt:60]: Ah good idea. I've also noticed that `Afterpay.configuration?.locale ?: Locale.US` is performed in multiple places so maybe it's worth consolidating it?

Comment (adamjcampbell) [reply afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt:60]: I thought this. I think it'll be something to address at the end once I've done currency formatting. The correct place to put it hopefully should be more apparent when everything is complete

Comment (Rypac) [reply afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt:60]: One thing you can do is create an extension on the nullable `Locale`, like is done with the `orEmpty()` extension on a nullable `List`:

```kotlin
internal fun Locale?.orDefault(): Locale = this ?: Locale.US
```

**Pull request merged**